day_7/ceasar_cipher.rb

```diff
@@ -0,0 +1,36 @@
+class CeasarCipher
+  def encode(text, shift)
```

Comment:
Nice work thinking in terms of classes and methods within those classes. For mod 1, you'll get into a pattern of using an `initialize` method and then abstracting the other methods to use.